ticket-issuer-url-website/src/App.js

```diff
@@ -148,12 +148,12 @@ function App() {
       <div className="flexCenter">
         <div className="tokensWrapper">
           {
-            tokens && tokens.length > 0 && tokens.map((tokenInstance, index) => {
+            tokens && tokens?.length > 0 && tokens.map((tokenInstance, index) => {
```

Comment:
This can be changed from 

tokens && tokens?.length > 0

to 

tokens?.length  > 0